### ISSUE TITLE
No module named 'quantdom.lib'

### ISSUE THREAD

**Issue by maesterbronn** [constverum/Quantdom#4]
```
Traceback (most recent call last):
  File "/usr/local/bin/quantdom", line 7, in <module>
    from quantdom.cli import cli
  File "/usr/local/lib/python3.6/site-packages/quantdom/__init__.py", line 29, in <module>
    from .ui import *  # noqa
  File "/usr/local/lib/python3.6/site-packages/quantdom/ui.py", line 10, in <module>
    from .lib import (  # noqa
ModuleNotFoundError: No module named 'quantdom.lib'

```

**Comment by ppop** [comment]
Not the best description of the error, for the future please be more descriptive.

I've made a [PR](https://github.com/constverum/Quantdom/pull/6) for this.

**Comment by matloobkhushi** [comment]
looks like a phishing tool; tried to install virus on your computer

**Comment by ppop** [comment]
what are you talking about exactly?

**Comment by constverum** [comment]
https://github.com/constverum/Quantdom/issues/7

**Comment by constverum** [comment]
Fixed in v0.1
